.travis.yml

```diff
@@ -1,63 +1,47 @@
 language: python
-sudo: required
-# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
-# validate at: http://lint.travis-ci.org/
-python:
-#  - "2.6"
-  - "2.7"
-#  - "3.2"
-#  - "3.3"
-#  - "3.4"
-#  - "3.5"
-#  - "3.5-dev" # 3.5 development branch
-#  - "nightly" # currently points to 3.6-dev
-# command to install dependencies
-cache:
-  directories:
-    - $HOME/.cache/pip/wheels
-    - $HOME/virtualenv/python2.7.9
-addons:
```

Comment:
I am not sure if any of those are really needed. Also, a few of those, like python-lxml, are not used since the env creation below will pull them again. By removing this and switching to `sudo: false` the build starts much quicker.